src/main/java/com/example/demo/Entity/Dress.java

```diff
@@ -0,0 +1,42 @@
+package com.example.demo.Entity;
+
+import com.example.demo.Enum.DressColor;
+import com.example.demo.Enum.DressSize;
+import com.example.demo.Enum.DressType;
+import jakarta.persistence.*;
+import lombok.*;
+
+@Data
+@Entity
+@Getter
+@Setter
+@AllArgsConstructor
+@NoArgsConstructor
+@Table(name = "dresses")
```

Comment:
```suggestion
@Table(name = "dress")
```
Name of the table should be singular

Reply:
ok